react/week4/github_user_search/src/consumeUser.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useEffect } from "react";
+import { UserSearchContext } from "./userSearch.js";
+
+const Api_url = "https://api.github.com/search/users?q=";
+
+const User = () => {
```

Comment:
since this is a component, it's good practice to name the file as such. You might wan't to name this file simply `User.js` (using capital U)

Reply:
Ok Kasper, i will do thank you.